cpp/oneapi/dal/algo/jaccard/detail/service.hpp

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+* Copyright 2020-2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include "oneapi/dal/algo/jaccard/common.hpp"
+
+namespace oneapi::dal::preview::jaccard::detail {
+
+ONEDAL_FORCEINLINE std::int32_t min(const std::int32_t &a, const std::int32_t &b) {
+    return (a >= b) ? b : a;
+}
+
+ONEDAL_FORCEINLINE std::int32_t max(const std::int32_t &a, const std::int32_t &b) {
+    return (a <= b) ? b : a;
+}
+
+ONEDAL_FORCEINLINE std::int64_t compute_number_elements_in_block(
```

Comment:
the lib use term 'count' but generally the name of the function might be much shorter